pkg/install/v1alpha1/cstor_volume_0.7.0.go

```diff
@@ -130,6 +131,32 @@ spec:
     - cstor-volume-list-listcstorvolumereplicacr-default-0.7.0
   output: cstor-volume-list-output-default-0.7.0
 ---
+# runTask to list cvrs if this is a clone volume
+apiVersion: openebs.io/v1alpha1
+kind: RunTask
+metadata:
+  name: cstor-volume-create-listclonecstorvolumecr-default-0.7.0
+spec:
+  meta: |
+    {{- $isClone := .Volume.isCloneEnable | default "false" -}}
+    id: cvolcreatelistclonecvr
+    runNamespace: {{.Config.RunNamespace.value}}
+    apiVersion: openebs.io/v1alpha1
+    kind: CStorVolumeReplica
+    action: list
+    options: |-
+    {{- if ne $isClone "false" }}
+      labelSelector: openebs.io/persistent-volume={{ .Volume.sourceVolume }}
+    {{- else }}
```

Comment:
can we avoid the else part completely. I did not understand the concept of listing resource using this label selector

Reply:
I will remove this. In case there are 1000s cvr and we try to list without selector then we would get flooded. To make it fast i introduced here a non existent label selectorm